### ISSUE TITLE
Consider changing domain from "rust-lang" to something less ambiguous.

### ISSUE THREAD

**Issue by LouisGariepy** [plabayo/learn-rust-101#5]
Regardless of trademark implications, I find that using "rust-lang" as a domain name implies some form of endorsement from the Project or the Foundation. Moreover, there is a definite element of confusion since Rust's official site also uses the "rust-lang" domain.

Furthermore, considering the guide advertises the author as a consultant and promotes donation links to them, I believe it should make a clear effort to distinguish itself from the official Rust website, and having the same domain name does not go towards that goal.

Do note that I'm not affiliated with the Rust Project and Foundation, I only offer this advice as someone with an interest in Rust learning material.

That said, feel free to close this issue without any changes. If you do, I'd appreciate if you could leave an explanation of your reasoning for keeping the domain name, for posterity.

Thanks.

**Comment by GlenDC** [comment]
Hi. Thank you for your interest. I understand where you’re coming from but I honestly don’t think there’s any possibility for ambiguity:

- there’s a clear disclaimer on github and on the front page of the book
- The GitHub repo is linked to on every page, which also will show you it’s not under any official Rust organisation
- I gave it the MIT license so you can honestly fork the guide, use it in whatever ways you want, etc…
- I am in no way affiliated with Rust Foundation, nor do I claim to be (anywhere)
- To this day I have received 0 donations for this book, or any other income made with it. There’s an option for it if you feel it’s worth something to you, given the many hours I’ve put into it and continue to keep relevant
  - If there’s a global request to remove the sponsorship or consult badges I’ll do with pleasure, I honestly do not mind either way but as I do think it’s fair to keep it there I’ll leave it for now (eg domains are also not free)
- I’m involved in and with Rust since early days and a big supporter, user and promoter of the project. This guide is my way to say thank you
- There are no affiliation links, tracking or any ways I make money from the free online guide
- It’s the guide I wish I had if I would start today, as honestly the landscape is big and not easy to know where to start and what’s out there
- This guide will be FOSS and gratis forever
- This repo is open to anyone to contribute. So far I’ve accepted all pull requests as is.

I have no bad intent and I believe this project is only good for Rust.

In case the foundation wants to take ownership of this project and use it as such I am open to transfer, completely for free. As long as I get attribution for the original versions and such I am more then fine with this.

I wish you well in life, thank you for your interest and have fun in your Rust learning journey and life in general.

**Comment by LouisGariepy** [comment]
@GlenDC Thanks for your response.

For the record, I did not mean to disparage your work or your character in any way shape or form. In fact it's quite the opposite: I thank you for your work on this guide and wish you the best in your endeavors.

What prompted me to write this issue is that I stumbled on this guide today (I made a tiny typo contribution a while back), and I was legitimately confused by the domain name, thinking it was an official Rust effort. I **do not** think at all  you're trying to take advantage of this situation, but I do legitimately believe there is a potential for confusion. IMO "rust-101" or something along those lines would be a more appropriate website name. That said, I respect your opinion on the matter and won't press this issue any further whatsoever.

I hope I'm being clear about the nature of my intentions.

Best wishes.
